demo/app/ios/Runner/Tests/IntegrationTest.swift

```diff
@@ -69,7 +69,9 @@ class IntegrationTest: XCTestCase {
         let vpInteraction = Openid4vpNewInteraction(openID4VPArgs, opts, nil)!
 
         let credentialsQuery = try vpInteraction.getQuery()
-        let inquirer = CredentialNewInquirer(nil)!
+
+        var newInquirerError: NSError?
```

Comment:
You're passing in an error now, but I don't see you checking it anywhere. So the error is still being ignored.